manifests/b/Bambulab/Bambustudio/1.8.4/Bambulab.Bambustudio.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created with YamlCreate.ps1 v2.3.3 $debug=QUSU.CRLF.5-1-22621-2506.Win32NT
+# yaml-language-server: $schema=https://raw.githubusercontent.com/microsoft/winget-cli/master/schemas/JSON/manifests/v1.6.0/manifest.installer.1.6.0.json
+
+PackageIdentifier: Bambulab.Bambustudio
+PackageVersion: 1.8.4
```

```suggestion
PackageVersion: 01.08.04.51
```
Please update file name and path to match this change.